src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+
+public class FigureSupplier {
+    ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
This variable should be private to follow the encapsulation principle. Also, consider whether this should be a class-level variable or local to the method where it's used.